content/development/best-practices.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Development Best Practices"
+description: ""
+slug: "development-best-practices"
+tags:
+  - "Best Practices"
+  - "Development"
+---
+{{< wip >}}
+
+![Terraform](/assets/08bcd99-terraform.png)
+
+# Use GitHub Workflow
+
+## Infrastructure as Code
+
+Infrastructure as Code is essential for managing change control and as a system of record for disaster recovery. Configuration is a form of Intellectual Property. Without the code, you don’t own the IP.
+
+## Feature Branches
+
+Recommend using a consistent naming convention for branch names by all developers. Using a convention will help developers to navigate branches.
+
+## Pull Requests
+
+Pull Requests should express `## what` changed and `## why` it changed (e.g. business justification). This forces contributors to be clear about what is happening and makes it easier for others to follow along.
+
+Attach a `terraform plan` against current master as a comment to all PRs.
```

Comment:
this probably needs more details on why against current master

Reply:
Going to just remove it.